crates/re_viewer/src/app.rs

```diff
@@ -1586,6 +1619,11 @@ fn open(app: &mut App) {
 
 #[cfg(not(target_arch = "wasm32"))]
 fn save(app: &mut App, loop_selection: Option<(re_data_store::Timeline, TimeRangeF)>) {
+    let Some(log_db) = app.log_db() else {
+        re_log::error!("No data to save!");
+        return;
+    };
```

Comment:
How can this ever happen? The `Save` button should be greyed out if there's no recording selected, right?

A comment would be nice to state this really shouldn't happen if that's the case.

Reply:
"Through the command palette" is the answer